pilot/pkg/serviceregistry/kube/client.go

```diff
@@ -68,18 +69,48 @@ func ResolveConfig(kubeconfig string) (string, error) {
 	return kubeconfig, nil
 }
 
-// CreateInterface is a helper function to create Kubernetes interface
-func CreateInterface(kubeconfig string) (*rest.Config, kubernetes.Interface, error) {
+// CreateInterfaceFromKubeconfig is a helper function to create Kubernetes interface from kubeconfig file
```

Comment:
Can we keep the old name ( and not change all other files )? We can discuss in a separate PR what is the best name for the method...

Reply:
Done.